src/storage/store/free_chunk_map.cpp

```diff
@@ -0,0 +1,177 @@
+#include "storage/store/free_chunk_map.h"
+
+
+#include "common/assert.h"
+#include "common/constants.h"
+#include "common/types/types.h"
+#include "main/client_context.h"
+#include "common/serializer/deserializer.h"
+#include "common/serializer/serializer.h"
+using namespace kuzu::common;
+
+namespace kuzu {
+namespace storage {
+
+FreeChunkMap::FreeChunkMap()
+    :maxAvailLevel(INVALID_FREE_CHUNK_LEVEL) {
+    /* we initialize our vector as nullptr vector by default */
+    for (int i = 0; i < MAX_FREE_CHUNK_LEVEL; i++) {
+        freeChunkList.push_back(nullptr);
+    }
+    maxAvailLevel = INVALID_FREE_CHUNK_LEVEL;
```

Comment:
Minor - is this line required, considering that the value is already initialized in the initializer list?